repos/fdbt-site/src/components/UserDataUploads.tsx

```diff
@@ -25,133 +25,264 @@ const UserDataUploadComponent = ({
     poundsOrPence,
     csrfToken,
     backHref,
```

Comment:
Why are we editing the UserDataUploads component? The change is only required to happen on the csvZoneUpload page